[@azure/monitor-query] Provide sovereign cloud support for monitor SDKs (#29871)
Azure/azure-sdk-for-js: sdk/monitor/monitor-query/README.md

#### Configure client for Azure sovereign cloud

> By default, all clients are configured to use the Azure public cloud. To use a sovereign cloud, provide the correct endpoint and audience value while instantiating the clients. For example:

Reviewer: ```suggestion
By default, the library's clients are configured to use the Azure Public Cloud. To use a sovereign cloud instead, provide the correct endpoint and audience value when instantiating a client. For example:
```

```ts
import { DefaultAzureCredential } from "@azure/identity";

> import { LogsQueryClient, MetricsQueryClient, metricsClient } from "@azure/monitor-query";

Reviewer: ```suggestion
import { LogsQueryClient, MetricsQueryClient, MetricsClient } from "@azure/monitor-query";
```
